spec/javascript/components/applications/__snapshots__/ListingApplicationsPage.test.js.snap

```diff
@@ -70,44 +70,48 @@ Array [
       className="loading-panel loading"
     >
       <div
-        className="spinner sk-fading-circle"
+        style={Object {}}
       >
         <div
-          className="sk-circle1 sk-circle"
-        />
-        <div
-          className="sk-circle2 sk-circle"
-        />
-        <div
-          className="sk-circle3 sk-circle"
-        />
-        <div
-          className="sk-circle4 sk-circle"
-        />
-        <div
-          className="sk-circle5 sk-circle"
-        />
-        <div
-          className="sk-circle6 sk-circle"
-        />
-        <div
-          className="sk-circle7 sk-circle"
-        />
-        <div
-          className="sk-circle8 sk-circle"
-        />
-        <div
-          className="sk-circle9 sk-circle"
-        />
-        <div
-          className="sk-circle10 sk-circle"
-        />
-        <div
-          className="sk-circle11 sk-circle"
-        />
-        <div
-          className="sk-circle12 sk-circle"
-        />
+          className="spinner sk-fading-circle"
```

Comment:
weird that we have the spinner in the snapshot, is it supposed to be loading? Probably not your fault if it is

Reply:
Looks like the spinner was there before this change, see line 73.

My guess is we're missing some combination of `await`, `update()`, or `tick` in the test. But I'm not going to change it right now...

Reply:
yep that seems reasonable